conda_recipe/u8timeseries/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set data = load_setup_py_data() %}
+
+package:
+  name: "u8timeseries"
+  version: {{ data.get('version').split('-', 1)[0] }}
+
+source:
+  # root folder, not the package
+  path: "../../../u8timeseries"
+
+build:
+  number: 0
+  script: '{{ PYTHON }} -m pip install . --no-deps -vv'
+
+requirements:
+  host:
+    - python >=3.6
+  run:
+  # only essential requirements to use this library ~ setup.py install_requires ~ requirements/main.txt
```

Comment:
Isn't there a way to tie this to `requirements/main.txt` ?

Reply:
Made it work :)

I can now do jinja 'coding', lolz